### PR TITLE
Remove import declaration when stripping flowtypes if flow specifiers

### PR DESCRIPTION
**Pull request by existentialism**
…f specifiers

| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | yes
| Major: Breaking Change?  | no
| Minor: New Feature?      | no
| Deprecations?            | no
| Spec Compliancy?         | no
| Tests Added/Pass?        | yes/yes
| Fixed Tickets            |
| License                  | MIT
| Doc PR                   |
| Dependency Changes       |

### CODE REVIEW

**Comment by hzoo** [comment]
Bug was found from https://github.com/babel/babel/blob/9e2ef0cf9e87b4b829d891d45102c96fca7c2038/packages/babel-plugin-transform-es2015-arrow-functions/src/index.js#L3 @Kovensky which before when stripped it resulted in `import "babel-traverse";` and https://unpkg.com/babel-plugin-transform-es2015-arrow-functions@7.0.0-alpha.10/lib/index.js 😓  haha.

File: packages/babel-plugin-transform-flow-strip-types/src/index.js [hunk 1]
```diff
@@ -19,6 +19,22 @@ export default function ({ types: t }) {
         }
       },
 
+      ImportDeclaration(path) {
+        if (!path.node.specifiers.length) return;
```
Comment (jridgewell) [inline packages/babel-plugin-transform-flow-strip-types/src/index.js]: Is this possible?

Comment (existentialism) [inline packages/babel-plugin-transform-flow-strip-types/src/index.js]: Was hoping to not drop cases of `import 'foo';`

Comment (hzoo) [inline packages/babel-plugin-transform-flow-strip-types/src/index.js]: Apparently so atm, `import {} from "babel-traverse";` parses.. but it turns into `import "babel-traverse";` basically

**Pull request merged**